src/hooks/index.js

```diff
@@ -1,5 +1,13 @@
+import useDribbbleShots from './useDribbbleShots';
 import useGetBreakpoint from './useGetBreakpoint';
 import useIsOpened from './useIsOpened';
-import useQueryStaticImages from './useQueryStaticImages';
+import useStaticImages from './useStaticImages';
+import useSiteMetadata from './useSiteMetadata';
```

Comment:
I understand now what you meant when you said that graphql hooks may not be suitable for the folder with other hooks.
I think it'll be better to either move them to the inner folder:
```
hooks/graphql/useSiteMetadata
hooks/graphql/useStaticImages
etc.
```
or to keep them in one file, e.g. `hooks/useGraphlQuery.js` with all queries in this file.

Reply:
I think that a directory will be better choice, because sometimes a query can be massive. I will move it.